.infrastructure/cronjob.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: todoapp-cronjob
+  namespace: mateapp
+spec:
+  schedule: "*/4 * * * *"
+  concurrencyPolicy: Allow
+  startingDeadlineSeconds: 200
+  successfulJobsHistoryLimit: 10
+  failedJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: todoapp
+            image: ikulyk404/busyboxplus:curl
+            args:
+            - sc
+            - c
```

Comment:
The `args` for the container seem to have a typo. The argument `sc` is not a valid command. Ensure that the command is correctly specified, possibly `sh` or another valid shell command.